src/jit/compiler.h

```diff
@@ -7612,6 +7612,18 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
     // Should we support SIMD intrinsics?
     bool featureSIMD;
 
+    // Should we recognize SIMD types?
+    // We always do this on ARM64 to support HVA types.
+    __declspec(property(get = GetSupportSIMDTypes)) bool supportSIMDTypes;
```

Comment:
I don't think new `__declspec(property)` should be added to the JIT...

Reply:
Yes, I suppose not. Believe it or not, I've been working on this for so long that I had added this before the (relatively) recent discussion of removing `__declspec`s.

Reply:
Ha ha, I believe you. Sadly I don't think it's currently possible to do changes in the ABI area of the JIT without a significant amount of time and care.

Reply:
> Sadly I don't think it's currently possible to do changes in the ABI area of the JIT without a significant amount of time and care

Right, plus, for this change I had to delve into the VM side, which I'm much less familiar with.